src/components/dialog/index.js

```diff
@@ -138,12 +138,12 @@ class Dialog extends BaseComponent {
   }
 
   renderContent() {
-    const {bottom} = this.getThemeProps();
+    const {bottom, testID} = this.getThemeProps();
     const bottomInsets = Constants.getSafeAreaInsets().paddingBottom;
 
     return (
       <TouchableWithoutFeedback>
-        <SafeAreaView style={{flexGrow: 1}}>
+        <SafeAreaView style={{flexGrow: 1}} testID={testID}>
```

Comment:
I would pass it to top possible View you can, I think in this case it's the one that is rendered in `renderAnimationContainer`

Reply:
Done